uptane-standard.md

```diff
@@ -905,7 +905,7 @@ To properly check Targets metadata, an ECU SHOULD:
 
 1. Download up to Z number of bytes, constructing the metadata filename as defined in {{metadata_filename_rules}}. The value for Z is set by the implementor. For example, Z may be tens of kilobytes.
 1. The version number of the new Targets metadata file MUST match the version number listed in the latest Snapshot metadata. If the version number does not match, discard it, abort the update cycle, and report the failure. (Checks for a mix-and-match attack.) Skip this step if checking Targets metadata on a partial verification ECU; partial verification ECUs will not have Snapshot metadata.
-1. Check that the Targets metadata has been signed by the threshold of keys specified in the relevant metadata file (Checks for an arbitrary software attack):
+1. Check that the Targets metadata has been signed by the threshold of keys specified in the relevant metadata file. (Checks for an arbitrary software attack.) Skip this step if checking Targets metadata on a partial verification ECU.
```

Comment:
Should we clarify that PV secondaries should check that the signature is good using public keys specified elsewhere (e.g., baked into ECU firmware)?

Reply:
How would that work? Wouldn't the only way to verify the signatures of the Targets metadata be to have the Root metadata available? That can't be baked in if it can be rotated.

Reply:
I think the understanding was always that PV secondaries would bake which Targets keys on the Director repository are currently trusted using the image itself. There is some allusion to this in the [key management](https://uptane.github.io/deployment-considerations/key_management.html) text (see recovering from a Director repository compromise, in particular), although we should make this clearer.

Is your understanding the same, @JustinCappos?

Reply:
Yes, this matches my understanding.  A PV secondary doesn't have a separate way to update its root of trust without updating the image.  If you wanted to add that and do it securely, you'd just make it a FV secondary...

Reply:
I see, I'd never realized that. So to rotate the Director root key, you have to bake it into a new image and send it down? Why is that better than sending the Root metadata along with the Targets metadata from the Primary? That still seems pretty far from full verification; you still have the benefit of not needing to send all the Image repo metadata and delegations.

Reply:
I'll retract my questions and objections here now that I better understand the intentions. I still think there is an interesting opportunity for fetching Root metadata as well as Targets as discussed on the call yesterday, but I'll move that discussion to a new issue.

So then I should just undo this change, right? A PV Secondary should have the keys specified in the Root metadata after all.